### PR TITLE
Zur 167 add members list for the channel members

### PR DESCRIPTION
**Pull request by YussaMoney**
> # Fixes Issue
> **My PR Fixes ID-167**
> 
> # Changes proposed
> ## What were you told to do ?
> **Add members list for channel member**
> 
> ### Additional Task
> **Fixed bug of not showing modal**
> 
> ## What did you do ?
> I [added member list for channel members 'ZUR-167'](https://github.com/zurichat/zc_main/pull/2625/commits/0bfd326b8da7ffd1b118790619e0f69863038e42)
 
> # Check List (Check all the applicable boxes)
> * [x]  My code follows the code style of this project.
> * [x]  This PR does not contain plagiarized content.
> * [x]  The title and description of the PR is clear and explains the approach.
> * [x]  I am making a pull request against the **dev branch** (left side).
> * [x]  My commit messages styles matches our requested structure.
> * [x]  My code additions will fail neither code linting checks nor unit test.
> * [x]  I am only making changes to files I was requested to.
> 
> # Screenshots/ Videos

> After I added my changes Site after changes

### CODE REVIEW

File: packages/main/src/auth/use-auth.js [hunk 1]
```diff
@@ -36,12 +36,14 @@ function useProvideAuth() {
       email,
       password
     });
-    const { data } = response.data;
+    const { data } = await response.data;
+    console.log(data.user.token);
```
Comment (JulianaSau) [inline packages/main/src/auth/use-auth.js:40]: Remove this console.log

**Review by JulianaSau — CHANGES_REQUESTED** [review]
Instead of showing dummy members on that modal, ensure that if the data is being fetched, i renders something like 'loading...' or no channel members yet.
All those channels have members. For a channel to exist, threr only has to be one member present so there's no need for the dummy data to appear

**Comment by YussaMoney** [comment]
Changes made

**Review by JulianaSau — CHANGES_REQUESTED** [review]
image
This code renders a dummy member list by default. edit this so that it doesnt show the dummy list. if the data is fetching, it should display a loader or indicate that the data is beiing fetched

**Comment by JulianaSau** [comment]
@YussaMoney, provide a video showing how it's working now

**Comment by JulianaSau** [comment]
> I don' t have a video

That just means go and screen record one and post it here. I need to see that it's working properly now before I merge

**Pull request closed without merging**